docs/memfault/Index.rst

```diff
@@ -0,0 +1,23 @@
+.. _index_memfault:
+
+Memfault Integration
+####################
+
+.. intro_start
+
+The `nRF Connect SDK <https://github.com/nrfconnect/sdk-nrf>`_ provides `integration with Memfault <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/others/memfault_ncs.html>`_. 
+
+By default the Memfault data is sent using HTTPs.
+Using MQTT is possible and preferred because it reduces the power consumption.
+The `nRF Asset Tracker Memfault integration for AWS IoT <https://github.com/NordicSemiconductor/asset-tracker-cloud-memfault-aws-js>`_ provides the neccessary cloud resources to send Memfault data using the already established MQTT cloud connection.
+Core dumps, which are collected when the application crashes, are still sent to Memfault using HTTPs this only happens once after a reboot and it allows to identify issues with the MQTT connection.
```

Comment:
```suggestion
By default, Memfault data is sent using HTTPs.
Using MQTT is possible and preferred, because it reduces power consumption.
The `nRF Asset Tracker Memfault integration for AWS IoT <https://github.com/NordicSemiconductor/asset-tracker-cloud-memfault-aws-js>`_ provides the neccessary cloud resources to send Memfault data using the established MQTT cloud connection.
Core dumps, which are collected when the application crashes, are still sent to Memfault using HTTPs.
This only happens once after a reboot and it allows you to identify issues with the MQTT connection.
```

Reply:
Should this be HTTPS instead?